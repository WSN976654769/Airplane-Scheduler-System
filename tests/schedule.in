location import locations.csv
flight import flights6.csv
schedule sydney
schedule perth
schedule beijing
schedule
schedule asd
schedule beijing perth
exit