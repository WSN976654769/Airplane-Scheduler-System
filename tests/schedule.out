User: Imported 51 locations.

User: Imported 169 flights.
13 lines were invalid.

User: Sydney
-------------------------------------------------------
ID   Time        Departure/Arrival to/from Location
-------------------------------------------------------
 153 Mon 10:30   Departure to Dubai
 100 Mon 12:00   Departure to Jakarta
 103 Mon 13:33   Arrival from AbuDhabi
  89 Mon 19:00   Departure to Hobart
  90 Mon 21:30   Departure to Hobart
 160 Tue 02:38   Arrival from AliceSprings
 161 Tue 04:00   Departure to AliceSprings
  69 Tue 05:00   Departure to LosAngeles
  99 Tue 08:27   Arrival from Hobart
   0 Wed 09:00   Departure to Beijing
   5 Wed 10:00   Departure to AbuDhabi
 148 Wed 11:00   Departure to LosAngeles
 149 Wed 13:00   Departure to Dubbo
 152 Wed 15:23   Arrival from Dubbo
 150 Wed 18:00   Departure to Orange
 151 Wed 20:14   Arrival from Orange
   6 Thu 04:44   Arrival from AbuDhabi
   3 Thu 11:11   Arrival from Hanoi
 125 Fri 14:00   Departure to AbuDhabi
 113 Sat 04:00   Departure to Perth
 114 Sat 06:00   Departure to Perth
 112 Sat 09:00   Departure to Perth

User: Perth
-------------------------------------------------------
ID   Time        Departure/Arrival to/from Location
-------------------------------------------------------
 132 Mon 00:00   Departure to Johannesburg
 158 Mon 15:05   Departure to AliceSprings
 157 Mon 16:05   Departure to AliceSprings
 156 Mon 17:52   Arrival from AliceSprings
 126 Fri 13:00   Departure to AbuDhabi
 113 Sat 08:31   Arrival from Sydney
 114 Sat 10:31   Arrival from Sydney
 112 Sat 13:31   Arrival from Sydney

User: Beijing
-------------------------------------------------------
ID   Time        Departure/Arrival to/from Location
-------------------------------------------------------
 129 Wed 20:00   Departure to Hanoi
   0 Wed 21:25   Arrival from Sydney
   1 Wed 22:25   Departure to Jakarta
   2 Wed 23:25   Departure to Hanoi
   4 Thu 00:25   Departure to Vladivostok
  24 Fri 18:16   Arrival from AbuDhabi
 128 Fri 20:00   Departure to Hanoi
 127 Fri 21:53   Arrival from Vladivostok

User: This location does not exist in the system.

User: This location does not exist in the system.

User: Beijing
-------------------------------------------------------
ID   Time        Departure/Arrival to/from Location
-------------------------------------------------------
 129 Wed 20:00   Departure to Hanoi
   0 Wed 21:25   Arrival from Sydney
   1 Wed 22:25   Departure to Jakarta
   2 Wed 23:25   Departure to Hanoi
   4 Thu 00:25   Departure to Vladivostok
  24 Fri 18:16   Arrival from AbuDhabi
 128 Fri 20:00   Departure to Hanoi
 127 Fri 21:53   Arrival from Vladivostok

User: Application closed.
